gen/installer/bash.py

```diff
@@ -172,6 +172,9 @@
   fi
 
   print_status $RC "Is SELinux disabled?"
+  if [[ $RC -ne 0 ]]; then
+    exit $RC
+  fi
```

Comment:
The rest of this script uses `OVERALL_RC` and checks cummulative return code in the caller. Exiting here will cause the preflight to stop when it hits this check instead of producing a summary (for subsequent checks)